RoadMap_SwiftUI/RoadMap_SwiftUI/View/DescriptionView.swift

```diff
@@ -0,0 +1,141 @@
+// DescriptionView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Экран описания товара
+struct DesriptionView: View {
+    // MARK: - Public property
+
+    var body: some View {
+        VStack {
+            topGradientView
+            furnitureItemImageView
+            selectedItemPriceRectangleView
+                .padding(.top, Constants.Numbers.paddingStandard)
+        }
+        .navigationBarBackButtonHidden()
+        .navigationBarTitleDisplayMode(.inline)
+        .offset(y: isTextEditing ? Constants.Numbers.keyboardOffset : Constants.Numbers.zeroFloat)
+    }
+
+    // MARK: - Private properties
+
+    @FocusState private var isTextEditing
```

Comment:
```suggestion
    @FocusState private var isTextEditing
    
```
Отступами разделяем разные враперы